src/v/cluster/commands.h

```diff
@@ -137,6 +138,12 @@ using move_partition_replicas_cmd = controller_command<
   model::record_batch_type::topic_management_cmd,
   serde_opts::adl_and_serde>;
 
+using move_topic_replicas_cmd = controller_command<
```

Comment:
I would change the commit message here a little bit since we are adding a new controller command